README.md

```diff
@@ -112,7 +123,9 @@ end
 Your answer:
 ```text
 
-Replace this with your answer
+Context and describe are functionally equivalent, but context is used specifically to nest tests within each other.
+TBH, I'm not sure how well this applies to the preceding example, but you could use context tags to test different
+units that make of different "branches" of the same shared "tree."
 ```
```

Comment:
In this example, `Describe` indicates what exactly is being tested -- the `add_tenant` method.
`Context` tells us the conditions under which the method is being tested.